src/Window.cpp

```diff
@@ -237,13 +243,28 @@ bool Gosu::Window::tick()
     }
     
     SDL_Event e;
+
     while (SDL_PollEvent(&e)) {
-        if (e.type == SDL_QUIT) {
-            SDL_HideWindow(shared_window());
-            return false;
-        }
-        else {
-            input().feed_sdl_event(&e);
+        switch (e.type) {
+            case (SDL_QUIT): {
+                SDL_HideWindow(shared_window());
+                return false;
+            }
+        #if SDL_VERSION_ATLEAST(2, 0, 5)
+            case (SDL_DROPBEGIN):
+            case (SDL_DROPCOMPLETE):
+            case (SDL_DROPTEXT):
```

Comment:
I'm on SDL 2.0.5 here and adding these additional event types changes nothing, it works just fine with just the SDL_DROPFILES block.